integration_tests/__tests__/snapshot-test.js

```diff
@@ -41,4 +49,74 @@ describe('Snapshot', () => {
       output['snapshot is not influenced by previous counter 0']
     ).not.toBe(undefined);
   });
+
+  describe('validates existance of snapshots correctly', () => {
```

Comment:
describe should usually only have a name, not a sentence. Like `describe('Validation')` and then the sentence that is built up will be `Snapshot > Validation > deletes the snapshot`.